Source/RuntimeAudioImporter/Private/RuntimeAudioTranscoder.cpp

```diff
@@ -117,6 +118,10 @@ void URuntimeAudioTranscoder::TranscodeRAWDataFromFile(const FString& FilePathFr
 
 		ExecuteResult(true);
 	}));
+#else
+	UE_LOG(LogRuntimeAudioImporter, Error, TEXT("file operation support disabled"));
```

Comment:
Could we add a more detailed description here? Such as: `UE_LOG(LogRuntimeAudioImporter, Error, TEXT("Unable to transcode RAW data from file '%s' to file '%s' because the file operation support is disabled"), *FilePathFrom, *FilePathTo);`.